TSSSF_CardGen.py

```diff
@@ -171,21 +171,22 @@
     }
 
 RulesDict={
-    "{replace}": "While in your hand, you may discard a Pony card from the grid and play this card in its place. This power cannot be copied.",
+    "{replace}": "While this card is in your hand, you may discard a Pony card from the grid and play this card in its place. This power cannot be copied.",
     "{swap}": "You may swap 2 Pony cards on the shipping grid.",
     "{3swap}": "You may swap up to 3 Pony cards on the grid.",
-    "{draw}": "You may draw a card from the Ship or Pony deck.",
-    "{goal}": "You may discard a Goal and draw a new one to replace it.",
-    "{search}": "You may search the Ship or Pony discard pile for a card of your choice and play it.",
+    "{draw}": "You may draw 1 card from the Ship or Pony deck.",
+    "{goal}": "You may discard 1 active Goal and draw 1 new Goal to replace it.",
+    "{search}": "You may search the Ship or Pony discard pile for 1 card of your choice and put it into your hand. If it’s still in your hand at the end of your turn, discard it.",
```

Comment:
Please remove the fancy apostrophes from the file as they may not print on all systems properly